pkg/importer/dataStream.go

```diff
@@ -899,10 +899,17 @@ func (d *DataStream) copy(dest string) error {
 	if util.GetAvailableSpace(d.ScratchDataDir) > int64(0) {
 		defer CleanDir(d.ScratchDataDir)
 	}
-	if d.isHTTPQcow2() {
-		err := d.convertQcow2ToRawStream(dest)
-		if err != nil {
-			return err
+	if d.isHTTPQcow2() || d.isIsoImage {
+		if d.isIsoImage {
+			err := StreamDataToFile(d.topReader(), dest)
```

Comment:
Did you check to see if there was enough space before streaming the data to the target file, like convertQcow2RawStream does in the validate?